### PR TITLE
Tuì-tsiàu pâi hōo tsê.

### PR DESCRIPTION
**Pull request by a8568730**
### Tsîng-hîng

Hāu-suán-sû kah tuì-tsiàu siunn uá, tsiok e̍h.

截圖 2020-11-26 上午11 43 54

### Àn-sǹg 
Khuànn hāu-suán-sû siōng tn̂g luā tn̂g, tī tuì-tsiàu thâu-tsîng thinn tab thinn hōo tsiâu.

### CODE REVIEW

**Comment by a8568730** [comment]
Makefile `tee build.log` kā tshò-ngōo tsia̍h--khì, hāi Travis-CI bē tú-tio̍h tshò-ngōo kā tòng--khí-lâi.
```
xcodebuild -project ThoanTaigi.xcodeproj -configuration Release build | grep -v setenv |  tee build.log
```
Huînn-ti̍t sī khò Travis-CI kiám-tsa, m̄-tsiah a28c1bc kui-khì kā `tee build.log` pak-tiāu.
```
xcodebuild -project ThoanTaigi.xcodeproj -configuration Release build
```

**Comment by a8568730** [comment]
截圖 2020-11-26 上午11 22 54

File: Makefile [hunk 1]
```diff
@@ -44,7 +44,8 @@ deps: librime
 
 release: $(DEPS_CHECK)
 	bash package/add_data_files
-	xcodebuild -project ThoanTaigi.xcodeproj -configuration Release build | grep -v setenv | tee build.log
+	xcodebuild -project ThoanTaigi.xcodeproj -configuration Release build
+
```
Comment (sih4sing5hong5) [inline Makefile:48]: 排版照本底，空tsuā thâi-thâi--leh

File: SquirrelPanel.m [hunk 2]
```diff
@@ -353,8 +353,59 @@ - (void)showPreedit:(NSString *)preedit
   NSRect highlightedRect = NSZeroRect;
   CGFloat separatorWidth = 0;
 
+
+
```
Comment (sih4sing5hong5) [inline SquirrelPanel.m]: 排版照本底，空tsuā thâi-thâi--leh

File: SquirrelPanel.m [hunk 3]
```diff
@@ -353,8 +353,59 @@ - (void)showPreedit:(NSString *)preedit
...
   // candidates
   NSUInteger i;
+
+  // For vertical mode, before appending candidates, copy the front workflow 
+  // of candidates to get the max width of candidates, so that the spacing 
+  // between candidate and comment can be calculated later.
```
Comment (sih4sing5hong5) [inline SquirrelPanel.m:361]: 這2tsuā後壁嘛有空白

Comment (a8568730) [reply SquirrelPanel.m:361]: bc5f980

File: SquirrelPanel.m (hunk 2 above)
Comment (a8568730) [reply SquirrelPanel.m]: bc5f980

File: Makefile (hunk 1 above)
Comment (a8568730) [reply Makefile:48]: bc5f980

**Review by sih4sing5hong5 — APPROVED** [review]

**Pull request merged**